downstream/assemblies/platform/assembly-installing-controller-operator.adoc

```diff
@@ -25,9 +25,10 @@ You can use these instructions to install the {ControllerName} operator on {OCP}
 
 == Installing the {ControllerName} operator
 
-include::platform/proc-install-aap-operator.adoc[leveloffset=+1]
+. Navigate to *Operators* > *Installed Operators*, then click on the *Ansible Automation Platform* operator.
+. Locate the *Automation controller* tab, then click *Create instance*.
 
-With the {PlatformName} operator installed, locate the *Automation controller* entry and click *Create instance*.
+You can proceed to configure the instance using either the Form View or YAML view.
```

Comment:
Suggestion: `You can proceed with configuring the instance using either the Form View or YAML view.`